### DISCUSSION TITLE
Reference Endnotes

### DISCUSSION THREAD

**Discussion by robertnwinter** [Zettlr/Zettlr#2923] [Q&A] [answered]
Hello All, hoping someone may be able to help with this. 

In Libre Writer, the Zotero plugin gives me the option to choose either Footnote or Endnote when using a note based citation style. 

In Zettlr, I know how to assign different styles to a project (APA, Oxford etc.), but I can't workout how to set note style citations to be endnotes when exported (it defaults to footnotes).

Any guidance would be most appreciated.

**Comment by nathanlesage** [comment, marked as answer]
This is possible by choosing appropriate footnote formats: https://pandoc.org/MANUAL.html#options-affecting-specific-writers

You can set the option `reference-location: document` to get endnotes at the end of your document. Alternatively, you can also set them to `block` which SHOULD result in the regular footnotes, or `section`, i.e. to place them at the end of a chapter. However, that does not work for PDF or OpenDocument. When exporting to OpenDocument or Word, you can change the display of the notes in the settings of the document, for PDF there currently is no way.

Here is a relevant issue on the Pandoc issue tracker: https://github.com/jgm/pandoc/issues/4041

**Reply by robertnwinter** [reply]
Many thanks for the response, most helpful. 

For the editorial process, I need to export to Word. Given that is a current limitation, will use the document settings approach to convert function to move footnotes to endnotes.
